src/main/java/com/nylas/AddVersionHeadersInterceptor.java

```diff
@@ -8,7 +8,7 @@
 
 public class AddVersionHeadersInterceptor implements Interceptor {
 
-	private static final String API_VERSION = "2.1";
+	private static final String API_VERSION = "2.2";
```

Comment:
Have you checked out what else changed in 2.2?  Anything we should watch out for?

Reply:
From what I gathered from the changelog, the only thing that the Java SDK supports that changes from 2.1 -> 2.2 is the addition of the new Room Resource fields: https://changelog.nylas.com/publications/api-version-2-2-now-available